synapse/lib/layer.py

```diff
@@ -3762,6 +3762,11 @@ async def iterEdgeVerbs(self, n1buid, n2buid):
         for lkey, venc in self.layrslab.scanByDups(n1buid + n2buid, db=self.edgesn1n2):
             yield venc.decode()
 
+    async def iterEdgePref(self, buid):
```

Comment:
How is this different than `iterNodeEdgesN1()` ( forgive me, i'm pre-coffee ;) )

Reply:
I figured the `edgesn1` store was gonna go away in v3 since we have the `n1n2` store and decided against hitting against it. If it's not, then yea, we can absolutely go back to using it.